CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+
+```shell
+make serve;
+```
+and then, go to [http://localhost:8484](http://localhost:8484)
+
+
+## Creating a new post
+
+Posts are stored under [`post` content directory](content/post)
+
+A blog post content stored under: `content/post/__POST_FILE_NAME__.md`
+
+will be accessible under: `//__BLOG_HOST_NAME__/post/__POST_FILE_NAME__`
+
+
+### Content schema
+
+Every blog post content file is stored under the path `content/post/__POST_FILE_NAME__.md`, and has the following schema:
+
+```
+---
+author: authorKeyName
+date: 2006-01-02
+title: "Post title"
+image: /post/__POST_FILE_NAME__/header-image.png
+description: "Short description of the post"
+categories: ["science", "technical", "culture"]
+draft: true
+---
+
+Whatever content in `markdown` format.
+
+```
+
+
+### Front matter
+
+The **front matter** is the first section inside the content file &ndash;enclosed by "`---`" chars&ndash; where it is defined the content metadata.<br />(Read more about it in the Hugo docs for the [Front Matter section](https://gohugo.io/content-management/front-matter))
+
+- `author`: Defines the authorship as explained in the [authors section](#authors)
+- `date`: Date when you plan to publish the post, `2006-01-02` format. **Do not forget to update it before publishing it.**
+- `title`: Post title, as it will appear in the very top of the content
+- `image`: Defines the content listing image as explained in the [assets section](#assets)
+- `description`: Short description of the post. Plain text only; no HTML nor markdown allowed.
+- `categories`: An array of at least one of the following: `science`, `technical`, `culture`. If it is not defined, it will not be served by the json blog api, so it will not appear in the source{d} landing.
+- `draft`: This key should not be present in public posts. Read more in the [peer review section](#peer-review)
+
+
+### Formatting the content
+
+It is not allowed to use HTML tags to format the content of the blog.
+
+If you want/need special formatting for your content, please read the [shortcodes tutorial](https://blog.sourced.tech/documentation/shortcodes)
+
+
+### Links to other source{d} blog posts
+
+Links from source{d} blog posts to other source{d} blog posts must be always relative to the blog url. That means that these urls will be like:
+```
+[link text](/post/__POST_FILE_NAME__)
+```
+
+
+### Assets
+
+Given a post content file stored under <br />
+`content/post/__POST_FILE_NAME__.md`,<br />
+its assets should be stored under :<br />
+`static/post/__POST_FILE_NAME__/__ASSET_FILE_NAME_PLUS_EXTENSION__`
+
+That way, the linkable urls would be:<br />
+`/post/__POST_FILE_NAME__/__ASSET_FILE_NAME_PLUS_EXTENSION__`
+
+When it is needed to serve demos with scripts and runnable stuff, the sources must not be hosted in the blog repository. To achieve these usecases, it should be used the [`codepen` shortcode](https://blog.sourced.tech/documentation/shortcodes#codepen)
```

Comment:
s/When it is needed to serve demos with scripts and runnable stuff, the sources must not be hosted in the blog repository/When demos or runnable stuff need resources, these must not be hosted in the blog repository, but externally./

Reply:
```
s/To achieve these usecases, it should be used the [`codepen` shortcode](https://blog.sourced.tech/documentation/shortcodes#codepen)/To do that, the [`codepen` shortcode](https://blog.sourced.tech/documentation/shortcodes#codepen) should be used./
```